OSDOCS-5239-RN: Documented the 4.14 quickly cluster install AWS LZ re… (#65012)
openshift/openshift-docs: release_notes/ocp-4-14-release-notes.adoc

==== Root volume types parameter for {rh-openstack} is now available
You can now specify one or more root volume types in {rh-openstack}, by using the `rootVolume.types` parameter. This parameter is available for both control plane and compute machines.

[id="ocp-4-14-quickly-install-cluster-aws-local-zones"]
==== Install a cluster quickly in AWS Local Zones

> For {product-title} {product-version}, you can quickly install a cluster on Amazon Web Services (AWS) to extend compute nodes to Local Zone locations. After you add zone names to the installation configuration file, the installation program fully automates the creation of required resources, network and compute, for each Local Zone.

Reviewer: > for each Local Zone.

Idk if it would be better to use "for" or "on".

> For {product-title} {product-version}, you can quickly install a cluster on Amazon Web Services (AWS) to extend compute nodes to Local Zone locations. By using this installation route, the installation program automatically creates network resources and Local Zone subnets for each Local Zone that you defined in your configuration file. 

Reviewer: I would remove duplicated LZ naming, and keep more generic (installer will not create only the subnet), adding the configuration introduced. Something like this:
```suggestion
For {product-title} {product-version}, you can quickly install a cluster on Amazon Web Services (AWS) to extend compute nodes to Local Zone locations. By using this installation route, the installation program fully automate the creation of required resources in Local Zones, network and compute, when the zone names are added in the installation configuration file.
```

I think we could explicity share 

 the installation program automatically creates network resources and Local Zone subnets for each Local Zone that you defined in your configuration file.

Author: Good point. I updated the suggestion with some slight changes. WDYT?

> For more information, see xref:../installing/installing_aws/installing-aws-localzone.adoc#installation-cluster-quickly-extend-workers_installing-aws-localzone[Intall a cluster quickly in AWS Local Zones]

Reviewer: Maybe?
```suggestion
For more information, see xref:../installing/installing_aws/installing-aws-localzone.adoc#installation-cluster-quickly-extend-workers_installing-aws-localzone[Intall a cluster quickly in AWS Local Zones].
```

Author: Well spotted. Updated!

> For more information, see xref:../installing/installing_aws/installing-aws-localzone.adoc#installation-cluster-quickly-extend-workers_installing-aws-localzone[Installing a cluster on AWS with worker nodes on AWS Local Zones]

Reviewer: You pointed to a quickly section, but not described it in the naming. I would added between brackets the name of the section. WDYT?
```suggestion
For more information, see xref:../installing/installing_aws/installing-aws-localzone.adoc#installation-cluster-quickly-extend-workers_installing-aws-localzone[Install a cluster quickly in AWS Local Zones]
```

Author: Good point. I added a more specific link. My original intention was to showcase the two routes, but RH AWS LZ users should already be familiar with the initial cluster installation route.
